.ci/pipeline_definitions

```diff
@@ -48,6 +48,12 @@ terraformer:
             target: terraformer
             build_args:
               PROVIDER: equinixmetal
+          terraformer-slim:
+            registry: 'gcr-readwrite'
+            image: 'eu.gcr.io/gardener-project/gardener/terraformer-slim'
+            target: terraformer
+            build_args:
+              PROVIDER: none
```

Comment:
```suggestion
              PROVIDER: slim
```

Reply:
Damn, how did I not see that one? Fixed it.